pkg/controllers/dynakube/kspm/token.go

```diff
@@ -0,0 +1,85 @@
+package kspm
+
+import (
+	"context"
+
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
+	"github.com/Dynatrace/dynatrace-operator/pkg/controllers/dynakube/kspm/consts"
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/dttoken"
+	k8ssecret "github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects/secret"
+	"k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func ensureKSPMSecret(ctx context.Context, client client.Client, apiReader client.Reader, dk *dynakube.DynaKube) error {
+	kspmSecretName := dk.Name + "-" + consts.KSPMSecretKey
```

Comment:
we will need this in the future, to mount it, should have a function either in this package called `GetSecretName` or as a method for the `DynaKube`

Reply:
done